packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/assetModelDataStreamExplorer/getAssetModelQueryInformation.ts

```diff
@@ -8,9 +8,14 @@ export const getAssetModelQueryInformation = (
 
   const assetModelId = firstAssetModel?.assetModelId;
   const assetId = firstAssetModel?.assetIds?.at(0);
-  const propertyIds = firstAssetModel?.properties.map(
-    ({ propertyId }) => propertyId
-  );
+
+  const propertyIds: string[] | undefined = firstAssetModel?.properties
+    .map(({ propertyId }) => propertyId)
+    .filter((propertyId, index) => {
+      // dedupe any new property ids
+      if (propertyIds) return propertyIds.indexOf(propertyId) === index;
```

Comment:
you can dedupe by creating a Set and then spreading that into an array:
which might be cleaner than this filter function

```
const propertyIds = firstAssetModel?.properties.map(
  ({ propertyId }) => propertyId
);

const uniquePropertyIds = [...new Set(propertyIds)]
```